src/components/HostSummary/HostSummary.jsx

```diff
@@ -0,0 +1,61 @@
+import styles from './HostSummary.module.css'
+
+function HostSummary({ hostName, hostingDuration, role, profilePicUrl }) {
+  const isSuperhost = role === 'Superhost';
+  return (
+    <div className={styles.hostInfo}>
+      <div className={styles.profilePicContainer}>
+        <img
+          src={profilePicUrl}
+          alt={`Hosted by ${hostName}`}
+          className={styles.profilePic}
+        />
+        {isSuperhost && (
```

Comment:
Instead of line 4, it's not better to simply write like this:
{role === "Superhost" && (
<svg
xmlns="http://www.w3.org/2000/svg"
viewBox="0 0 12 14"...

Reply:
Thank you for the comment. For me, it is simpler to make the code reusable. Instead of writing role === 'Superhost' in many places, we can just use isSuperhost. Currently, it may seem redundant since I've only used it twice, but if we add more conditional logic that utilizes isSuperhost in the future, it could be useful.  So, I see no problem to use it as is or delete const isSuperhost = role === 'Superhost'; and change isSuperhost to role === 'Superhost'.